### ISSUE TITLE
Running the creator using apimtemplate release 

### ISSUE THREAD

**Issue by apurvc** [Azure/azure-api-management-devops-resource-kit#211]
Hi,

I have two questions on using the creator utility as suggested in the documentation.
![Screenshot 2019-07-11 at 17 14 34](https://user-images.githubusercontent.com/3213275/61067471-81222f80-a3ff-11e9-9b1e-9da9e3e3a176.png)

1. How exactly to use the creator utility? do we need to add this in path? though I tried and it didn't work.
2. How to setup this utility on Mac?

**Comment by apurvc** [comment]
@miaojiang please can you suggest what can be done.

**Comment by apurvc** [comment]
I figured out how to install and execute this utility this is high level what I have done.

1. download and unzip the utility.
2. Inside the publish directory a binary with the name "**apimtemplate**" is present 
3. Put this location in your path or reference it like below from where the file is present
e.g. ~/Downloads/publish/./apimtemplate create --configFile xyx.yaml

To run it on Mac I installed dotnet-sdk-2.2.301-osx-x64.pkg 

Probably a clear instructions to use could be better.
